packages/amplify-e2e-tests/src/__tests__/cdk/backends/base-cdk/app.ts

```diff
@@ -2,7 +2,7 @@
 import 'source-map-support/register';
 import { App, Stack, Duration } from 'aws-cdk-lib';
 // @ts-ignore
-import { AmplifyGraphqlApi, AmplifyGraphqlDefinition } from '@aws-amplify/graphql-construct-alpha';
+import { AmplifyGraphqlApi, AmplifyGraphqlDefinition } from '@aws-amplify/api-graphql-construct';
```

Comment:
Should the package name be `@aws-amplify/graphql-api-construct` to align with the import name `AmplifyGraphqlApi`?

Reply:
@renebrandel to confirm, but this is approved by leadership specifically.

Reply:
Yeah, we'll flip to `graphql-api-construct`. I'll update that in the next rev.